src/router/index.js

```diff
@@ -4,49 +4,58 @@ import BallotDrop from "@/components/BallotDrop";
 
 Vue.use(VueRouter)
 
-
-  const routes = [
-    {
-      path: '/',
-      name: 'Home',
-      component: BallotDrop,
-      props: {
-      }
+const routes = [
+  {
+    path: '/',
+    name: 'Home',
+    component: BallotDrop,
+    meta: {
+      title: 'Home - Ballot Drop',
     },
+  },
   {
     path: '/about',
     name: 'About',
     // route level code-splitting
     // this generates a separate chunk (about.[hash].js) for this route
     // which is lazy-loaded when the route is visited.
-    component: () => import(/* webpackChunkName: "about" */ '../views/About.vue')
+    component: () => import(/* webpackChunkName: "about" */ '../views/About.vue'),
+    meta: {
+      title: 'About - Ballot Drop',
+    },
   },
     {
       path: '/contact',
       name: 'Contact',
       // route level code-splitting
       // this generates a separate chunk (about.[hash].js) for this route
       // which is lazy-loaded when the route is visited.
-      component: () => import(/* webpackChunkName: "contact" */ '../views/Contact.vue')
+      component: () => import(/* webpackChunkName: "contact" */ '../views/Contact.vue'),
+      meta: {
+        title: 'Contact - Ballot Drop',
+      },
     },
     {
       path: '/:state',
       name: 'State',
       component: BallotDrop,
       props: {
         state_route: null
-
-      }
+      },
+      meta: {
+        title: route => `${route.params.state} - Ballot Drop`,
```

Comment:
Can you replace the `-` with a space in the state name, so `New-Jersey` becomes `New Jersey`? nitpicky but while you're in there!